TUTORIAL.md

```diff
@@ -0,0 +1,100 @@
+# Getting started
+Before you start, make sure you understand the concept or database migration and rollback.
```

Comment:
Rephrase to "make sure you understand the concept of forward/up and reverse/down database migrations"

Reply:
Renamed